Source/FullScreenPokemon.ts

```diff
@@ -3788,31 +3785,31 @@ module FullScreenPokemon {
                     return output;
                 })
             });
-            FSP.MenuGrapher.setActiveMenu("Pokedex");
+            this.MenuGrapher.setActiveMenu("Pokedex");
 
-            FSP.MenuGrapher.createMenu("PokedexOptions");
-            FSP.MenuGrapher.addMenuList("PokedexOptions", {
+            this.MenuGrapher.createMenu("PokedexOptions");
+            this.MenuGrapher.addMenuList("PokedexOptions", {
                 "options": [
                     {
                         "text": "DATA",
                         "callback": function (): void {
-                            FSP.openPokedexListing(
+                            this.openPokedexListing(
                                 currentListing.title,
-                                FSP.MenuGrapher.setActiveMenu.bind(FSP.MenuGrapher, "PokedexOptions"));
+                                this.MenuGrapher.setActiveMenu.bind(this.MenuGrapher, "PokedexOptions"));
                         }
```

Comment:
You'll need to convert this `function (): void { ... }` to `(): void => { ... }`. The former does not preserve scope, which means `this` will mean something else in side it. The latter does.

...and you'll have to fix this everywhere else :smile: have fun!